Testscripts/Windows/FCOPY-DISABLE-ENABLE.ps1

```diff
@@ -52,8 +52,7 @@ function Main {
     $BuildNumber = Get-HostBuildNumber $HvServer
     if ($BuildNumber -eq 0) {
         return "FAIL"
-    }
-    elseif ($BuildNumber -lt 9600) {
+    } elseif ($BuildNumber -lt 9600) {
```

Comment:
the line 58 below should be inside of 'else' block?

Reply:
no need, since if enter into if or elseif, the following code won't be executed.